### PR TITLE
feat: gcp implementation

### PR DESCRIPTION
**Pull request by luan-asym**
# Summary

> Adds scripts to extract FHIR resources from PFB extract and inserts them into GCP

## Pull Request Type

- [x] New feature

## Changelog

- _added_  `.ndjson` uploader script
- _added_ `.json` splitter script. The script is not currently used
- _added_ demo notebook to test scripts
- _updated_ PFB extractor script

## Additional Notes

- These scripts are not currently on GCP, but will be added soon

### CODE REVIEW

File: gcp/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# AnVIL GCP Integration
+
+> NodeJS script that takes an `.avro` upload and loads the data into Google's Healthcare API
+
+## Goals
+
+[ ] Create a script that puts test data in Google Healthcare API
+[ ] Transform script to accept an `.avro` export and insert into Google Healthcare API
+[ ] Deploy script to run off bucket trigger on GCP (may require App Engine)
```
Comment (bwalsh) [inline gcp/README.md:9]: + [ ] Deploy IG to Google Healthcare API
+ [ ] Review service w/ Broad devops & security teams

**Review by bwalsh — CHANGES_REQUESTED** [review]

File: gcp/README.md (hunk 1 above)
Comment (luan-asym) [reply gcp/README.md:9]: what does IG mean?

Comment (bwalsh) [reply gcp/README.md:9]: `IG` ~ "Implementation Guide"
* https://github.com/ncpi-fhir/ncpi-fhir-ig  
* https://www.hl7.org/fhir/implementationguide.html

**Review by bwalsh — APPROVED** [review]
Thanks

File: gcp/README.md (hunk 1 above)
Comment (luan-asym) [reply gcp/README.md:9]: ahh thank you. google was no help with that one

**Review by bwalsh — COMMENTED** [review]
Looks great.  Should we merge now? (My assumption is that there will be more scripts to setup & load the Healthcare API)

**Comment by luan-asym** [comment]
The changes are ready for merge. I'm exploring a way to get all of this automated.

**Pull request merged**